protobuf_definitions/message_formats.proto

```diff
@@ -384,6 +423,22 @@ message Reference {
   float altitude = 7; // Reference used in auto altitude mode (m)
 }
 
+/*
+ * Reference for the auto pilot when a mission is active.
+ */
+message ReferenceAutoPilot {
+  uint32 active_path_segment_id = 1; // id of the active path segment
```

Comment:
Could maybe be the path object directly, depending on the logic, i.e. whether the sender known the state of the whole path object.